libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -571,7 +571,7 @@ pub mod tests {
     #[test]
     fn test_heartbeat_notifications_raw_tcp() {
         let mut transport = Box::new(RawTransport::default());
-        let mut listener = transport.listen("tcp://localhost:8080").unwrap();
+        let mut listener = transport.listen("tcp://localhost:3030").unwrap();
```

Comment:
These could be `:0` which will assign a random port (and not conflict).  See the transport tests as an example